src/modules/YaPI/Samba.pm

```diff
@@ -17,11 +17,6 @@ $serverRole = DetermineRole()
   returns a string representing the currently configure role or
   undef on failure.
 
-$enabled = GetServiceStatus()
-
-  returns true if the services smbd and nmbd are enabled, false if either of
-  them is disabled or undef on failure.
-
 $enabled = EditService($enable)
```

Comment:
here it use previous $enable

Reply:
maybe mark in doc that the YAPI is deprecated?

Reply:
> here it use previous $enable

And also in other methods.

Probably we should, but what to do with old testsuite? Drop them completely or adapt them

Reply:
well, then in this example use e.g. true here.

I am not sure about old testsuite. Can we keep it without big investments? if so, then I would keep it, otherwise I think it is fine to drop it.